docs/testing.rst

```diff
@@ -1,5 +1,6 @@
 Tests
 ========
+In order to run test which rely on the TZPRO API you have to copy the `.env.example` file to `.env` and add your personal TZPRO API Key.
```

Comment:
for .env you would need to load it in with the function dot_env(). 
Just say that you need to set the environmental variable `export TZPRO_API_KEY="XXXXXXXXXX"` before running the tests. If not some tzpro dependent test might fail.

Reply:
For vscode you can put:

```
"python.testing.pytestArgs": [
    "--env",
    "TZPRO_API_KEY=XXXXXXXXXX",
    "tests"
],
```

in .vscode/settings.json